packages/vue/src/stories/getting-started/tailwind-css.stories.mdx

```diff
@@ -0,0 +1,295 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { YouTube } from 'mdx-embed';
+
+<Meta title="Setup/Integrations/TailwindCSS" />
+
+# TailwindCSS
+
+[TailwindCSS](https://tailwindcss.com/) is a utility-first CSS framework packed with classes like flex, pt-4, text-center and rotate-90 that can be composed to build any design, directly in your markup.
+You can use the it to customize **StorefrontUI** components in your application.
+
+<YouTube youTubeId="uCxPb_39TNg" />
```

Comment:
Is it possible to make it smaller? Currently this video holds almost whole screen.